addons/cni-cilium/helm-values

```diff
@@ -20,3 +11,23 @@ operator:
 
 preflight:
   enabled: false
+
+serviceAccounts:
+  hubblecertgen:
+    create: false
+  relay:
+    create: false
+  ui:
+    create: false
+
+hubble:
```

Comment:
Just to check, have you updated https://github.com/kubermatic/kubeone/blob/54b50e3ed51ebced1d9973263673918df5e641d5/addons/cni-cilium/hubble.yaml?